### ISSUE TITLE
Image tagging queue

### ISSUE THREAD

**Issue by abandoned-prototype** [lucyparsons/OpenOversight#821]
When tagging officers faces in images we give our users three options

1. Identify an officer, find the id and enter it in the form. Click "Add identified face" and be brought back to the image to identify more, or choose another option
2. Click "skip" - which will keep the image in the identification queue, but will randomly select another image (although it could be the same) from the queue
3. Click "Done with this image" - the image will be labeled as "tagged"=Done and no longer appear in the queue

I went through the images sitting in the CPD-queue in the last few days and noticed that there are a lot of images in there. A few new ones but also some that were uploaded in 2017. Basically all pictures had officers in them, but many had them with either unreadable or covered name-tags and star-numbers, or just a common last name that was legible. Some also only contained officers that were members of a different agency and a few could be identified but are retired now and are not yet included in the database.

So I am not sure if this has been discussed before, but the problem I saw was that the queue contained so many basically useless images that the whole process felt pretty boring. Especially seeing images that have been uploaded years ago and probably been looked at quite a few times by different people.

I ended up clicking "Done with this image" on all images I could not identify anyone in, to empty the queue and make it immediately show newly uploaded images. Now I am hoping to come up with a solution so this manual emptying doesn't have to be done. For example an image being skipped a certain number of times being set as "Done" after that, or at least the chance of an image being randomly selected decreasing the longer it has been sitting in the queue.

A somewhat related problem are images with a clearly visible face of an officer, but no identification. Meaning together with a different image it could be tagged, but there is no guarantee such an image exists, so you can either click "Skip" and hope that someone else might recognize the officer later (leading to the queue growing) or "Done" basically sending the image into the void.

One more issue I found is that an image that has one officer clearly identified and another not identifiable. A volunteer will potentially correctly identify the first officer and then click "Skip", hoping that someone else might be able to identify the other officer, however the next person seeing the image will again try to tag the first officer and either seeing the image on the officers profile or getting an error message after submitting the selected face making the whole process less efficient and more frustrating. To avoid this particular problem we could provide a list of officers already identified whenever loading an image to be tagged.

Finally, I think it would be good to have some kind of overview per department to see all images that have been marked as complete which is only accessible for admins and area coordinators. This would allow those users to find images that have been marked "Done" incorrectly or give them the ability to go through and find officers they might recognize.

So I want to start a conversation about this whole process to see if we can identify some smaller fixes that can help with the issues above (or others), or if we want to perhaps overhaul the whole system and for example making it more robust against intentional or unintentional incorrect actions.

**Comment by rot13x2** [comment]
I agree with everything you said here. An additional issue (that you touched on) is that the CPD-queue contains many images of officers from other departments, such as Berkeley, Oakland, and NYPD. That's likely because the default department for image submissions is CPD - combined with images auto-submitting once uploaded, without user confirmation, inadvertent uploads to wrong departments seem pretty likely. An easy solution would be to modify the volunteer interface to allow taggers to move images to different departments, but that could just clutter the volunteer interface too much, especially combined with some of your other suggestions above.
